pkgs/racket-doc/scribblings/reference/stx-trans.scrbl

```diff
@@ -474,7 +478,11 @@ can use bindings previously introduced into @racket[intdef-ctx].
 
 @defproc[(syntax-local-bind-syntaxes [id-list (listof identifier?)]
                                      [expr (or/c syntax? #f)]
-                                     [intdef-ctx internal-definition-context?])
+                                     [intdef-ctx internal-definition-context?]
+                                     [extra-intdef-ctxs (or/c internal-definition-context?
+                                                              (listof internal-definition-context?)
+                                                              #f)
```

Comment:
Is it worth allowing `#f` in addition to `null`? I ave no objection if it seems better to allow `#f` for consistency with other functions that historically accept `#f`, although the "extra"ness of this argument seems different.